src/qtiItem/core/Loader.js

```diff
@@ -142,8 +142,32 @@ var Loader = Class.extend({
                 }
 
                 //important : build responses after all modal feedbacks and outcomes has been loaded, because the simple feedback rules need to reference them
+                let responseRules = data.responseProcessing && data.responseProcessing.responseRules
+                    ? [...data.responseProcessing.responseRules]
+                    : [];
                 for (let i in data.responses) {
-                    const response = this.buildResponse(data.responses[i]);
+                    const responseIdentifier = data.responses[i].identifier;
+                    const responseRuleItemIndex = responseRules.findIndex(({ responseIf: {
+                        expression: {
+                            expressions: [expression = {}] = [],
+                        } = {}
+                    } = {} }) => expression.attributes
+                        && expression.attributes.identifier === responseIdentifier
+                        || (
+                            expression.expressions
```

Comment:
Second case with `expression.expressions[0].attributes.identifie` means that can be such object?
`{responseIf: { expression: { expressions: [ expressions: [{attributes:{identifier:...}}]]}}`

Reply:
yes, it is the case for `MAP_RESPONSE` and `MAP_RESPONSE_POINT ` templates.
templates can be found here src/qtiItem/helper/responseRules.js